lib/charms/layer/jenkins/configuration.py

```diff
@@ -99,3 +100,12 @@ def _set_prefix(self, prefix):
             return True
 
         return False
+
+    def set_update_center_ca(self):
+        """Configure Jenkins Update Center CA cert"""
+        config = hookenv.config()
+        ca_cert = config["update-center-ca"]
+        ca_cert_file = os.path.join(paths.UPDATE_CENTER_ROOT_CAS,
+                                    "default.crt")
+        host.mkdir(paths.UPDATE_CENTER_ROOT_CAS, owner="jenkins", group="jenkins", perms=0o750)
+        host.write_file(ca_cert_file, ca_cert, owner="jenkins", group="jenkins", perms=0o644)
```

Comment:
Same here, I would point to the file path being written (too much information can't hurt during debug).

Reply:
Juju already logs when using host.write_file():
```
2021-01-12 19:04:02 DEBUG juju-log Writing file /var/lib/jenkins/update-center-rootCAs/default.crt jenkins:jenkins 644
```